bin/fail2ban-client

```diff
@@ -430,6 +430,9 @@ class ServerExecutionException(Exception):
 	pass
 
 if __name__ == "__main__": # pragma: no cover - can't test main
+	from time import strptime
+	# strptime thread safety hack-around - http://bugs.python.org/issue7980
+	strptime("2012", "%Y")
```

Comment:
ha -- I think I had considered a similar hack at some point but don't remember why I haven't PRed it

shouldn't it be sufficient just to do once in ```fail2ban/__init__.py```.  that one I believe is imported by all the tools and we don't thread in the tools explicitely